zeroex/orderwatch/order_watcher_test.go

```diff
@@ -840,23 +824,35 @@ func TestOrderWatcherOrderExpiredThenUnexpired(t *testing.T) {
 		orderopts.SetupMakerState(true),
 		orderopts.ExpirationTimeSeconds(expirationTimeSeconds),
 	)
-	blockwatcher, orderWatcher := setupOrderWatcher(ctx, t, ethRPCClient, meshDB)
+	blockwatcher, orderWatcher := setupOrderWatcher(ctx, t, ethRPCClient, database)
 	watchOrder(ctx, t, orderWatcher, blockwatcher, ethClient, signedOrder)
 
 	orderEventsChan := make(chan []*zeroex.OrderEvent, 2*orderWatcher.maxOrders)
 	orderWatcher.Subscribe(orderEventsChan)
 
 	// Simulate a block found with a timestamp past expirationTime
-	latestBlock, err := meshDB.FindLatestMiniHeader()
+	latestBlocks, err := database.FindMiniHeaders(&db.MiniHeaderQuery{
```

Comment:
Considering how often this is used throughout the codebase, I think we should just add a utility method for it in the `db` package.

Reply:
👍 Good idea. I added a helper method and a test for it.